GameGraphic.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GameOfLife;
+using System;
+
+namespace GameOfLife
+{
+    public class GameGraphic : Game
+    {
+        private Game gameOfLife;
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="gameObject"></param> game object that will be made 
+
+        public GameGraphic(Game gameObject)
+        {
+            gameOfLife = gameObject;
+            Console.CursorVisible = false;
+        }
+
+        /// <summary>
+        /// Draws the game board. 
+        /// </summary>
+        public void DrawBoard()
+        {
+            for (int i = 0; i < gameOfLife.Height; i++)
+
+                for (int j = 0; j < gameOfLife.Width; j++)
+                {
+                    if (gameOfLife.currentGeneration[i, j] == 1)
+                        Console.Write("*");
```

Comment:
Having one code statement after if or for is valid, but may cause errors, so in professional code there always should be block containing one or many statements. Like:
if (gameOfLife.currentGeneration[i, j] == 1)
{
     Console.Write("*");
 }
 
 Please change where appropriate.